modules/update-mco-process.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this machine configuration update, control plane nodes and compute nodes are organized into their own machine config pools, where the pools of machines are updated in parallel to each other. The `.spec.maxUnavailable` configuration, which has a default value of one node, determines how many nodes in a machine config pool can simultaneously undergo the update process at any given time.
```

Comment:
control plane and compute node > control plane node and compute node

So both instances do not gel into one noun string.

Reply:
s/in parallel to each other/in parallel   
Currently, it seems redundant.

s/`.spec.maxUnavailable` configuration/`.spec.maxUnavailable` parameter or attribute 
(we really need a single term for these things!)

s/a default value of one node/a default value of `1`  
This is a real nit, as the parameter doesn't _really_ take _one node_ as a value.

s/at any given time/at the same time ??